views/dash.html

```diff
@@ -6,7 +6,10 @@ <h2>Groups</h2>
   <li>
     <a href='{{ url_for('get_group', groupid=group.id) }}'>Management page</a> |
     <a href='{{ url_for('get_events', groupid=group.id) }}'>Events page</a> |
-    {{ group.name }}
+    {{ group.name }} |
+    <a href='{{ url_for('del_group', groupid=group.id) }}'>
+      <button type="submit">Delete Group</button>
```

Comment:
We can leave the button part I think since the api is actually just a GET url. Were you trying to style the element in a button like fashion? If that is the case we can leave out the `submit` part and leave the button there.

The submit part makes it look like it is part of a form, which is not the case.